_posts/2020-05-30-newpipe-0.19.4-release.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "NewPipe 0.19.4 released: Stream fixes, SoundCloud comments, Restricted Mode setting, and our own F-Droid repository"
+short: "NewPipe 0.19.4 released"
+date: 2020-05-30 00:00:00 +0100
+categories: [release, pinned]
+author: gkeegan
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Keep on streaming with a fresh update to NewPipe. This one comes with a lot of bug fixes, improvements and more features. This also fixes an error when trying to watch YouTube videos.
+
+<!-- more -->
+
+## Additions
+
+Thanks to [@B0pol](https://github.com/B0pol), there is now support for SoundCloud comments! Go see if they recommend any similar music to your favorite track!
+
+There is now a Restricted Mode setting for YouTube. This will help keep some of the weird stuff on YouTube away. Thanks, [@vnagel](https://github.com/vnagel)!
+
+Viewing parent channel of a channel is now possible when watching a video of the sub-channel. This allows you to navigate easily to the channel that is related to the channel whose video you are watching. That's a lot of channels, so thanks [@Royosef](https://github.com/Royosef).
+
+## Bug fixes
+
+The bug where trying to open a video made a "Couldn't get the any stream" toast notification appear was fixed by [@wb9688](https://github.com/wb9688) in the extractor.
+
+They also fixed sending e-mails from the crash reporter, so next time you get a crash and choose to report it via e-mail, the error will be in the contents! This change will help errors get noticed by the developers faster.
+
+The "open in browser" button now actually opens the link in your default browser instead of giving an option of what to open it with. That was fixed by [@B0pol](https://github.com/B0pol).
+
+Another contribution by [@wb9688](https://github.com/wb9688) was to stop a freeze that happened when pressing "Download" on a PeerTube video.
+
+[@B0pol](https://github.com/B0pol) fixed some crashes related to different languages being set. Hindi speakers and Polish users need not worry about NewPipe crashing because of their language anymore.
+
+## Improvements
+
+Speaking of translations, the debug settings (for if you are running a test build of NewPipe) are now translateable. Help out with translating with [Weblate](https://hosted.weblate.org/projects/newpipe/strings/). Thank you [@wb9688](https://github.com/wb9688) for helping people around the world use NewPipe!
+
+The extractor version was updated by [@Stypox](https://github.com/Stypox), bringing better performance. [@TobiGr](https://github.com/TobiGr) made a commit related to that as well.
+
+NewPipe now uses vector drawables instead of PNGs for the icons in the app. This decreased the app size in general and allowed removing some legacy libraries. Great work [@Stypox](https://github.com/Stypox)! Icons should now scale very well.
```

Comment:
>and allowed removing some legacy libraries

No, we had to upgrade to non-legacy preferences though, because the legacy preferences doesn't support vector drawables, however I would've upgraded that otherwise anyway.